sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -142,7 +142,7 @@ prediction or predictions about a single text input. An operation's result type
 about the input document and how it was processed. An operation result contains a `isError` property that allows to
```

Comment:
line 141 (I can't comment on it) has a reference to `text input`.

Reply:
updated